### PR TITLE
Configure from email from Site object

### PR DESCRIPTION
**Pull request by joshblum**
Addresses #26

### CODE REVIEW

File: registration/models.py [hunk 1]
```diff
@@ -25,6 +26,22 @@
 SHA1_RE = re.compile('^[a-f0-9]{40}$')
 
 
+def get_from_email(site=None):
+    if getattr(settings, 'REGISTRATION_USE_SITE_EMAIL', False):
```
Comment (dicato) [inline registration/models.py]: We should probably add a docstring here.

**Comment by dicato** [comment]
This looks straightforward. Are there any manual testing steps?

File: registration/models.py (hunk 1 above)
Comment (joshblum) [reply registration/models.py]: done

**Comment by joshblum** [comment]
No manual testing tests, just the added unit tests

**Pull request merged**